docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -53,58 +49,68 @@ interval_month = YYYY-MM
 interval_year = YYYY
 ```
 
-There is also experimental support to use Browser locale/language to dynamically change the current date format
-for each user. This feature is disabled by default as it needs more testing and refinement.
+There is also experimental support to use the browser location and language to dynamically change the current date format for each user. This feature is disabled by default.
+
+The [Configuration]({{< relref "../administration/configuration.md#date_formats" >}}) topic has been updated as a result of this change.
+
+## Field options are out of beta!
+
+After lots of testing and user feedback, we removed the beta label from the configuration options in the Field and Override tabs. This release also includes the following feature enhancements.
 
-## Table column filters
+### Added table column filters
 
-The new column filter option allows you to dynamically apply value filters to any column. This option is disabled but
-can be enabled for all or a specific column using an override rule.
+You can now dynamically apply value filters to any table column. This option is can be enabled for all columns or one specific column using an override rule.
 
 {{< docs-imagebox img="/img/docs/v72/table_column_filters.png" max-width="800px" caption="Table column filters" >}}
 
-## New and improved transforms
+### New field override selection options
 
-Grafana 7.2 includes a new group by transform that allows you to group by multiple fields and add any number of aggregations for other fields.
+You can now add override rules that use a regex matcher to choose which fields to apply rules to.
 
-There is also an update to the labels to fields transform that allow you to pick one label and use that as the name of the value field.
+The [Field configuration options]({{< relref "../panels/field-configuration-options.md" >}}) content has been updated as a result of these changes.
```

Comment:
```suggestion
The [Field options]({{< relref "../panels/field-options.md" >}}) content has been updated as a result of these changes.
```

Renamed this page to Field options as we agreed in the docs channel :) https://github.com/grafana/grafana/pull/27604